### PR TITLE
feat(tutorial): React step 5 - Boe

### PR DESCRIPTION
**Pull request by baraaabdelsalam**
Closes #

http://carbon-tutorial-baraaabdelsalam.mybluemix.net/

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/StaticFile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+root: build
```
Comment (ZrianinaMariia) [inline src/StaticFile]: I believe you can delete this file.

File: src/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (ZrianinaMariia) [inline src/manifest.yml]: Also you don't need additional manifest in src/ folder.

**Comment by ZrianinaMariia** [comment]
@baraaabdelsalam Almost there! I would kind ask you to delete couple of unnecessary files (see review). 

Feel free to stop your running application on IBM Cloud to avoid the possibility of incurring any charges.

**Comment by smanikandan-btech** [comment]
hi @ZrianinaMariia  and @baraaabdelsalam  May I now how to create the pull request? i'm can't raise the pull request. if you please give me the steps to create the pull request. it will me very much helpful for me. I have raised my issue here - #2429 please check once.

Thanks in advance.

**Comment by ZrianinaMariia** [comment]
@baraaabdelsalam Looks fantastic 🥇! I've recorded your completion of step 5.

**Pull request closed without merging**